lib/calendarium-romanum/cli.rb

```diff
@@ -11,11 +11,23 @@ class CLI < Thor
     def query(date_str=nil)
       I18n.locale = options[:locale]
 
-      data_file = Data[options[:calendar]]
-      if data_file.nil?
-        die! 'Invalid calendar. See subcommand `calendars` for valid options.'
+      # This block is for specifying a full filesystem path
+      if File.exist?(options[:calendar])
+        begin
+          sanctorale = SanctoraleLoader.new.load_from_file(options[:calendar])
+        rescue CalendariumRomanum::InvalidDataError
+          die! 'Invalid file format.'
+        end
+      else
+        # This block is for predefined calendars
+        data_file = Data[options[:calendar]]
+
+        if data_file.nil?
+          die! 'Invalid calendar. See subcommand `calendars` for valid options.'
```

Comment:
If the user makes an error while entering calendar path
`calendariumrom query --calendar /path/with/a/typo`
the program will respond `Invalid calendar. See subcommand `calendars` for valid options.` The error message should take into account that it is not really clear if the user was trying to load a bundled calendar or custom calendar from a filesystem path.